requirements/dev-requirements.txt

```diff
@@ -47,7 +47,7 @@ beautifulsoup4>=4.11.1
 # NLP
 nltk<=3.6.7
 datasets
-langdetect
+fasttext
```

Comment:
remove `fasttext` from this file
it is already in the `nlp-prop-requirements.txt`